src/pages/molecules/itemCartBtn/helper/helperFunctions.js

```diff
@@ -0,0 +1,44 @@
+import produce from "immer";
+
+export const incrementItemHandler = function incrementItemHandler(itemIndex, setState){
+    setState(produce(draft => {
+        draft.vegetableItems[itemIndex].count += 1;
+    }));
+}
+
+export const decrementItemHandler = function decrementItemHandler(itemIndex, setState){
+    setState(produce(draft => {
+        draft.vegetableItems[itemIndex].count -= 1;
+    }));
+}
+
+export const removeItemHandler = function removeItemHandler(itemIndex, setState){
```

Comment:
Why have we nor written
const removeItemHandler=()=>{
..
}

or 
function removeItemHandler (itemIndex, setState){
.....
}